Assets/Scripts/Player/PlayerRaycasts.cs

```diff
@@ -0,0 +1,54 @@
+﻿using UnityEngine;
```

Comment:
Same as enemy raycasts, I would try and move most of this stuff to the classes where it is used (mostly player movment)